src/utils/serializers.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable camelcase */
+import { Serializer } from 'cerealizr';
+
+const bookSerializer = new Serializer({
+  descriptor: {
+    image_url: 'imageUrl'
+  },
+  mapAllValues: true
+});
+
+export const serializeBook = book => book && bookSerializer.serialize(book);
```

Comment:
beware that if book is an empty object, your book component will crash

Reply:
Yes, you are right! Good eye!